src/components/dataset-query/dataset-query.js

```diff
@@ -57,17 +58,30 @@ const DatasetQuery = props => {
     </span>
   )
 
+  const recommendationSection = () => {
+    const toolTipText = 'These datasets have related fields or columns that may be suitable for joining in your query'
+    if (recommendations && recommendations.length > 0) {
```

Comment:
I would say that lodash's `isEmpty` is a more expressive way of doing this check and I'd prefer it - Ben